src/descriptor/bare.rs

```diff
@@ -20,7 +20,7 @@
 
 use std::{fmt, str::FromStr};
 
-use bitcoin::{self, blockdata::script, Script};
+use bitcoin::{self, blockdata::script, Address, Network, Script};
```

Comment:
I was not sure what to do about this. I saw, as you say, this is done a bunch. I get it for `PublicKey` and `PrivateKey` but I don't get it for `Network`/`Address` etc. since this is a bitcoin library we all know what these things are. I was also unsure because of uniformity in this crate vs uniformity across the stack, I'm not exactly sure but I think it makes sense to try for uniformity across the stack?

Is there some perspective on using `bitcoin::Address` that I'm missing. Its not the first time I've debated removing paths from identifiers so I know I tend to err on the side of too terse :)

Reply:
I do this because I work a lot on rust-elements (which has its own fork of miniscript which uses the new opcodes) so it's easy to distinguish `elements::Address` from `bitcoin::Address`. For `Address` in particular it also helps me distinguish a "bitcoin address" from a "network address" or "memory address" etc.

I agree that within rust-miniscript itself, this serves little purpose. I mildly prefer things the way they are but don't have strong feelings either way.

Reply:
> I'm not exactly sure but I think it makes sense to try for uniformity across the stack

Yep, I would like to be consistent throughout the stack. I think with all the new contributors and throughout the lifetime of the project, this seems difficult to enforce/guarantee. 

I agree that using `Address` makes sense (although there is also IP address in the network module). I always have had the personal convention to `external_crate::DataStructure` or have exactly one thing before the `::` in the path. I don't have a strong rationte to do this, just a habit I subconsciously formed.

Reply:
> I do this because I work a lot on rust-elements (which has its own fork of miniscript which uses the new opcodes) so it's easy to distinguish elements::Address from bitcoin::Address

For the purposes of having minimal conflicts when porting to elements-miniscript, it might be better to `Addresss` instead of `bitcoin::Address`. Considering this, if you don't have any preference between the two I would rather it be `Address` than `bitcoin::Address` as it would save me some time when porting changes from here.

Reply:
Ok cool, good feedback. Let's go with `Address` (and `Network`) then.